govc/dvs/create.go

```diff
@@ -49,9 +51,12 @@ func (cmd *create) Register(ctx context.Context, f *flag.FlagSet) {
 	cmd.DVSCreateSpec.ConfigSpec = cmd.configSpec
 	cmd.DVSCreateSpec.ProductInfo = new(types.DistributedVirtualSwitchProductSpec)
 
+	cmd.numUplinkPorts = 2 // Set Default Uplink ports to 2
```

Comment:
I think the default should be '0' to maintain the existing behavior, what do you think?

Reply:
My only concern around that is someone reading the code will see cmd.numUplinkPorts = 0 and assume that this creates zero ports when it actually skips the if block and creates 2 uplinks.

Reply:
I think it's ok to leave it as: `f.UintVar(&cmd.numUplinkPorts, "num-uplinks", 0, "Number of Uplinks")`
And let the `CreateDVS` method do what it does by default.